valusets/Valuesets-England-OperationOutcomeCodes.xml

```diff
@@ -22,15 +22,15 @@
     <compose>
```

Comment:
Is this valuset required in your branch? If required, kindly use the updated version in [OperationOutcome branch](https://github.com/NHSDigital/NHSEngland-FHIR-ImplementationGuide/blob/Feature/IOPS-1512-Profiling-England_OperationOutcome/valusets/Valueset-England-OperationOutcomeCodes.xml)

Reply:
Copied OperationOutcome file